kolena/annotation.py

```diff
@@ -412,13 +412,19 @@ class ScoredLabeledTimeSegment(TimeSegment):
 @dataclass(frozen=True, config=ValidatorConfig)
 class TextSegment(Annotation):
     """
-    Represents a segment of text within a specified text field. The `start` index is inclusive and the `end` index is
-    exclusive, following the convention of Python string slicing.
+    Represents a segment of text within a specified text field to highlight. The `start` index is inclusive and the `end` index is
+    exclusive, following the convention of Python string slicing. The following example adds highlights to two text fields
+    named `text` and `summary`.
 
     ```py
+    text = "Hello, world"
+    summary = "A summary of current documentation."
+
     text_segments: List[TextSegment] = [
-        TextSegment(text_field="text", start=0, end=5),
-        TextSegment(text_field="summary", start=10, end=51),
+    # Highlights 'Hello' in 'Hello, world'
```

Comment:
Are we sure that this is configured correctly? This doesn't seem to line up from the python side:
```
>>> text = 'Hello, world'
>>> summary = "A summary of current documentation."
>>> text[0:6]
'Hello,'
>>> summary[2:13]
'summary of '
```

Reply:
I am not sure actually :) I guess thats why this example is useful. I will change to above.

Reply:
probably cleanest to adjust the indices to `0, 5` and `2, 12`